src/monitor/node_active_protocol.c

```diff
@@ -108,6 +110,11 @@ register_node(PG_FUNCTION_ARGS)
 
 	checkPgAutoFailoverVersion();
 
+	if (!PG_ARGISNULL(4))
```

Comment:
Yes that's why I got it back to being STRICT finally, and use zero instead of NULL to show that we don't have the system identifier yet. Not ideal, but allowed to solve all the other things to solve in that PR, might revisit later.

This is a left-over.

Reply:
Fixed now.